packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.1]
+### Changed
+- Add dependencies `eth-query` and `babel-runtime` ([#1447](https://github.com/MetaMask/core/pull/1447))
+-
+### Fixed
+- Fix bug where `query` function failed to call built-in EthQuery methods ([#1447](https://github.com/MetaMask/core/pull/1447))
```

Comment:
This bug didn't relate to this controller. That PR just improved the type of an internal method, effectively having no impact on the package beyond adding a dependency